doc/descriptors.md

```diff
@@ -119,6 +127,22 @@ steps, or for dumping wallet descriptors including private key material.
 
 In order to easily represent the sets of scripts currently supported by
 existing Bitcoin Core wallets, a convenience function `combo` is
-provided, which takes as input a public key, and constructs the P2PK,
-P2PKH, P2WPKH, and P2SH-P2WPH scripts for that key. In case the key is
-uncompressed, it only constructs P2PK and P2PKH.
+provided, which takes as input a public key, and matches P2PK,
+P2PKH, P2WPKH, or P2SH-P2WPH scripts for that key. In case the key is
+uncompressed, it only matches the P2PK or P2PKH scripts.
+
+### Output Descriptors vs. Addresses
+
+Like a bitcoin address, a bitcoin output descriptor can be used as a human
```

Comment:
I think this is not a similarity but a difference: an address does not (generally) convey information on how to spend it; a descriptor does (unless `raw` or `addr` are used).

Reply:
Yeah, this sentence would probably be clearer dropping "that a given wallet knows how to spend" from the end. The difference you are describing is basically difference #1 in the list below.

But I think it might be a good idea to drop this section entirely. Basically I was trying to describe the properties of descriptors by contrasting them with addresses. But I think it might be better to have a section just describing some ways descriptors can be used (internally or externally), since this is something actually I'm fuzzy on.

Reply:
I suspect this will clear itself up when more uses for descriptors appear.